x/liquidity/spec/02_state.md

```diff
@@ -1,105 +1,117 @@
 
 ## Pool
 
-`Pool` defines the liquidity pool that contains pool information
+The Pool state stores information about the pool type, reserve coin denoms, reserve address, and denom of the liquidity pool.
+
+The Pool state stores static information of a liquidity pool in this structure:
 
 ```go
 type Pool struct {
-	Id                     uint64         // index of this liquidity pool
-	TypeId                 uint32         // pool type of this liquidity pool
-	ReserveCoinDenoms      []string       // list of reserve coin denoms for this liquidity pool
-	ReserveAccountAddress  string         // reserve account address for this liquidity pool to store reserve coins
-	PoolCoinDenom          string         // denom of pool coin for this liquidity pool
+    Id                     uint64         // index of this liquidity pool
+    TypeId                 uint32         // pool type of this liquidity pool
+    ReserveCoinDenoms      []string       // list of reserve coin denoms for this liquidity pool
+    ReserveAccountAddress  string         // reserve account address for this liquidity pool to store reserve coins
+    PoolCoinDenom          string         // denom of pool coin for this liquidity pool
 }
 ```
 
-Pool: `0x11 | Id -> amino(Pool)`
+The parameters of the Pool state are:
 
-PoolByReserveAccIndex: `0x12 | ReserveAcc -> nil`
+- Pool: `0x11 | Id -> amino(Pool)`
 
-GlobalLiquidityPoolIdKey: `[]byte("globalLiquidityPoolId")`
+- PoolByReserveAccIndex: `0x12 | ReserveAcc -> nil`
 
-ModuleName, RouterKey, StoreKey, QuerierRoute: `liquidity`
+- GlobalLiquidityPoolIdKey: `[]byte("globalLiquidityPoolId")`
 
-PoolCoinDenomPrefix: `pool`
+- ModuleName, RouterKey, StoreKey, QuerierRoute: `liquidity`
+
+- PoolCoinDenomPrefix: `pool`
 
 ## PoolBatch
 
-`PoolBatch` defines the batch(es) of a given liquidity pool that contains indexes of deposit / withdraw / swap messages. Index param increments by 1 if the pool id is same.
+The PoolBatch state stores information about the target liquidity pool, the batch index, the height at the beginning of the batch, the last index of the messages, and batch execution status.
+
+The PoolBatch state stores information about the liquidity pool batch in this structure:
 
 ```go
 type PoolBatch struct {
-	PoolId           uint64  // id of target liquidity pool
-	Index            uint64  // index of this batch
-	BeginHeight      uint64  // height where this batch is begun
-	DepositMsgIndex  uint64  // last index of DepositMsgStates
-	WithdrawMsgIndex uint64  // last index of WithdrawMsgStates
-	SwapMsgIndex     uint64  // last index of SwapMsgStates
-	Executed         bool    // true if executed, false if not executed yet
+    PoolId           uint64  // id of target liquidity pool
+    Index            uint64  // index of this batch
+    BeginHeight      uint64  // block height when batch is created
```

Comment:
@dongsam  you'll see i suggested changes for the comments in the code, too

Reply:
sure, I'll apply it all at once just before merging this PR